File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.cs
public class Solution {
    public int MaxProfit(int[] prices) {
        int maxProfit = 0;
        int i = 0; int j = i+1;
        while (j < prices.Length)
        {
            if (prices[i] < prices[j])
            {
                maxProfit = Math.Max(maxProfit, (prices[j] - prices[i]));
                j++;
            }
            else
            {
                i = j;
                j++;
            }
        }

        return maxProfit;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxCount = 0
        count = 0
        for n in nums:
            if n == 1:
                count+=1
                continue
            maxCount = max(maxCount, count)
            count = 0
        maxCount = max(maxCount, count)
                
        
        return maxCount
